Sources/Keyboard/KeyboardKey.swift

```diff
@@ -36,6 +37,7 @@ public struct KeyboardKey: View {
         self.whiteKeyColor = whiteKeyColor
         self.blackKeyColor = blackKeyColor
         self.pressedColor = pressedColor
+        self.flatLeading = flatLeading
```

Comment:
I'm really not loving the addition of flatLeading to thiis.

Reply:
We can keep `flatTop` meaning top of the key is flattened.

Reply:
I use `alignment = .trailing` to determine if the key is vertical for now, until we have a case when label should be aligned to leading meaning the end of the keys is pointing to the left? Right now the top of the key is always on the left  and end is on the right in vertical layout. Not sure if someone needs it flipped.

Reply:
Updated padding logic like this:

https://github.com/AudioKit/Keyboard/blob/ee75112dc58fbbe44a1c04fae97ad1b7e15c22cf/Sources/Keyboard/KeyboardKey.swift#L89-L90

Meaning we are flattening the top if we align to bottom, we are flattening the leading if we align to trailing, and vice versa.